keyboards/4pplet/ibe60/keymaps/via/rules.mk

```
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Has no configured encoders
```suggestion
VIA_ENABLE = yes
```